skeleton/aws/modules/vpc/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "namespace" {
+}
```

Comment:
please provide the description & default type. This is the template, so the more detailed, the better, for any users of the template

```suggestion
variable "namespace" {
  description = "The namespace for the VPC"
  type = string
}
```

Reply:
Added in https://github.com/nimblehq/infrastructure-templates/pull/27/commits/fe3318a48516b380ea6b34a5347ede9d6a35cfa0